skt/executable.py

```diff
@@ -77,20 +77,21 @@ def report_results(result_path, result_string, report_path, report_string):
         report_file.write(report_string)
 
 
-def remove_oldresult(output_dir, prefix_filename):
+def remove_oldresult(output_dir, prefix):
     """
     Remove existing results from previous runs.
 
     Args:
-        output_dir:      Source directory stores existing result.
-        prefix_filename: Prefix of the existing result filename.
+        output_dir: Source directory storing existing result.
+        prefix:     Prefix of the existing results to be removed.
     """
-    try:
-        for filename in os.listdir(output_dir):
-            if filename.startswith(prefix_filename):
+    for filename in os.listdir(output_dir):
+        if filename.startswith(prefix):
+            try:
                 os.unlink(join_with_slash(output_dir, filename))
-    except OSError:
-        pass
+            except OSError:
```

Comment:
It will probably be OK, but it might be better to check for `os.errno.EISDIR` and re-raise everything else. What do you think?

Reply:
Works for me!